style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: black;
+}
```

Comment:
- [ ] Kindly add a class to `<h1>` as requested by the previous reviewer and give it your favorite color instead of giving it to h1 tag.